README.md

```diff
@@ -23,8 +23,11 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
 | [AddCode](https://addcode-io.breezy.hr) [:rocket:](https://addcode-io.breezy.hr/#positions) | Software engineering. | `Porto` |
+| [Affinity](https://affinity.pt) [:rocket:](https://affinity.pt/en/careers/) | Consultancy, Outsourcing. | `Lisbon` `Porto` |
```

Comment:
Can you please replace `Lisbon` with `Lisboa`?  
This also applies to the other entries.
Thanks!